src/Domain/Exceptions/IException.php

```diff
@@ -0,0 +1,7 @@
+<?php declare(strict_types=1);
+namespace DDDWorkshop\Domain\Exceptions;
+
+interface IException
```

Comment:
I don't recommend creating an interface for exceptions. The exception above is good enough